index.js

```diff
@@ -157,6 +156,23 @@ function marshalArray(arrayData) {
     return arrayData.length === 1 ? arrayData[0] : arrayData;
 }
 
+/**
+ * Joins a list of file paths as sass import statements.
+ *
+ * @param {string|string[]} files - file paths
+ * @returns {string}
+ * @private
+ *
+ * @example
+ * const joined = joinSourceFiles(['file1.scss', 'file2.scss']);
+ * console.log(joined); // "@import 'file1.scss';\n@import 'file2.scss';"
+ */
+function joinSourceFiles(files) {
```

Comment:
@phillipluther No concerns with this method of concatenation before compilation? Originally, I was reading and concatenating the contents of each sass file, we decided not to go that direction for fear of unpredictable results. Any concerns we could have similar issue with this approach?

Reply:
RE: Reduce ... I landed in the same place. I was tinkering with it last night (when I got frustrated thinking about the CLI stuff!) and started refactoring as a single reduce. It was incredibly difficult to read. Pretty code counts.

RE: Concatenation ... no concerns. I was thinking mostly about scenarios where stuff from one file might overwrite stuff from another file, and whether we go `@import` or simply concat raw contents its the same.

I'm pretty sure it's the same thing, actually - because `node-sass` will import and _then_ compile, not compile then concat. So if we were to compile each file individually it would change the fundamental behavior.

Reply:
Walking back on this a bit - the CLI doesn't support globs, but does have that `-r` flag for compiling an entire directory ... similar, but not the same. So if we add glob support, is that really the only thing our flavor of the CLI will enhance?

And maybe something akin to `--importers` or `--functions` for doing filename or output transformations?

Reply:
We should move this conversation to #4. I have posted a handful of thoughts/concerns/questions there over the past few weeks. 

In addition to globs and the config file, our CLI would also offer the other features unique to our Node API: compiling multiple files into one, dynamically changing the output filename `.min.css`, as well any other unique features we create (transforming file contents, interpolation patterns, etc.). However, I am skeptical about being able to include those features while keeping the same API as the current CLI.